server/tests/conftest.py

```diff
@@ -156,18 +165,18 @@ def tpm3_tx_t_element(tpm3_gene):
 
 
 @pytest.fixture(scope="module")
-def tpm3_tx_g_element(tpm3_descriptor):
+def tpm3_tx_g_element(tpm3_gene):
     """Provide TranscriptSegmentElement for TPM3 gene constructed using genomic coordinates and
     gene name.
     """
     return {
         "type": "TranscriptSegmentElement",
         "transcript": "refseq:NM_152263.4",
         "exonStart": 6,
-        "exonStartOffset": 5,
+        "exonStartOffset": 72,
         "exonEnd": 6,
-        "exonEndOffset": -71,
-        "gene": tpm3_descriptor,
+        "exonEndOffset": -5,
+        "gene": tpm3_gene,
```

Comment:
this example changed quite a bit after the change. The exon starts and ends and offsets seemed to have swapped. Unsure if this was intended, would def love some eyes on this from someone who understands these more than I do

Reply:
This may have been because of offset bug in cool-seq-tool which was fixed. I can look into this, but in the following lines we should have start or end not both in a sequence location when looking at Transcript Segment Elements

Reply:
I think the values are off in the Sequence Locations too

Reply:
oop you're right, good catch. Let me see if I can figure out what's going on there

Reply:
I think this might be the same issue in FUSOR tests where only some values are being checked? If so, we can just create a separate issue. Just as long as the actual output never has both start/end for transcript segment element

Reply:
added some checks for this!